### PR TITLE
Add heists and a slight modification to payday

### PR DESCRIPTION
**Pull request by Naleksuh**
Credit to BlueMageX for the code (I got his permission to pr it)

### CODE REVIEW

**Comment by Kowlin** [comment]
Thanks for putting in the effort to make a PR. However as this change includes files from a 3rd party and include changes that remove other fixes and changes from Economy. And there for I don't think we can accept this PR. However, I'll leave it open for @Twentysix26 to decide. I'll add more conflicting stuff in a review.

File: cogs/economy.py [hunk 1]
```diff
@@ -398,21 +360,8 @@ def __init__(self, bot):
         except NoAccount:
             await self.bot.say("User has no bank account.")
 
-    @_bank.command(pass_context=True, no_pm=True)
```
Comment (Kowlin) [inline cogs/economy.py:401]: Deleting a new command.

File: cogs/economy.py [hunk 2]
```diff
@@ -281,22 +235,31 @@ def __init__(self, argument):
 
 
 class Economy:
-    """Economy
+    """Economy - Blue variant
```
Comment (Kowlin) [inline cogs/economy.py:238]: "Blue variant" has to be removed in order to get this mainstreamed for other bots

File: cogs/economy.py [hunk 3]
```diff
@@ -629,60 +956,159 @@ def already_in_list(self, accounts, user):
...
         server = ctx.message.server
         self.settings[server.id]["PAYDAY_TIME"] = seconds
-        await self.bot.say("Value modified. At least {} seconds must pass "
-                           "between each payday.".format(seconds))
+        await self.bot.say("**Payday cooldown:** " + str(seconds) + " seconds")
         dataIO.save_json(self.file_path, self.settings)
 
     @economyset.command(pass_context=True)
-    async def paydaycredits(self, ctx, credits: int):
+    async def paydaycredits(self, ctx, creds: int):
```
Comment (Kowlin) [inline cogs/economy.py:1035]: Unneeded variable rename(?)

File: cogs/economy.py [hunk 4]
```diff
@@ -629,60 +956,159 @@ def already_in_list(self, accounts, user):
...
+##        """List shop items or buy an item!"""
+##        if ctx.invoked_subcommand is None:
+##            msg = "```"
+##            for k, v in self.shoplist:
+##                msg += "{}: {}\n".format(k, v)
+##            msg += "```"
+##            await self.bot.say(msg)
+
     def display_time(self, seconds, granularity=2):
-        intervals = (  # Source: http://stackoverflow.com/a/24542445
```
Comment (Kowlin) [inline cogs/economy.py:682]: Removing comments and source kills off readability

**Review by Kowlin — CHANGES_REQUESTED** [review]

**Comment by Twentysix26** [comment]
I suggest turning it into a separate cog if you want to share it with the community

File: cogs/economy.py (hunk 2 above)
Comment (Naleksuh) [reply cogs/economy.py:238]: Ah yes I forgot about that

**Pull request closed without merging**